README.md

```diff
@@ -6,43 +6,29 @@ Composer is a new programming model from [IBM
+OpenWhisk](https://github.com/apache/incubator-openwhisk).
 
-The [IBM Cloud functions shell](https://github.com/ibm-functions/shell)
-offers a CLI and graphical interface for fast, incremental, iterative,
-and local development of serverless apps. Some additional highlights
-of the shell include:
+## Getting started 
+* [Introduction to Serverless Composition](docs/README.md): Setting up your programming environment and getting started with Shell and Composer 
+* [Building a Serverless Translate Bot with Composition](docs/translateApp.md): A more advanced tutorial that describes using Composition to build a serverless Slack chatbot that does language translation. 
+* [Learning more about Composer](docs/COMPOSER.md), a Node.js module to author compositions using JavaScript
 
-* Edit your code and program using your favorite text editor, rather than using a drag-n-drop UI.
-* Validate compositions with readily accessible visualizations, without switching tools or using a browser.
-* Deploy and invoke compositions using familiar CLI commands.
-* Debug invocations with either familiar CLI commands or readily accessible visualizations.
+## Videos
+* [IBM Cloud Shell YouTube channel](https://www.youtube.com/channel/UCcu16nIMNclSujJWDOgUI_g): The channel hosts demo videos of IBM Cloud Shell, including editing a composition [using a built-in editor](https://youtu.be/1wmkSYl7EDM) or [an external editor](https://youtu.be/psqoysnVgE4), and [visualizing a composition's execution](https://youtu.be/jTaHgDQDZnQ). 
+* Watch [our presentation at Serverlessconf'17](https://acloud.guru/series/serverlessconf/view/ibm-cloud-functions) about Composition and Shell 
 
-Composer and shell are currently available as IBM Research
-previews. We are excited about both and are looking forward to what
-compositions you build and run using [IBM Cloud
-Functions](https://ibm.biz/openwhisk) or directly on [Apache
-OpenWhisk](https://github.com/apache/incubator-openwhisk).
+## Example applications
+* A _Serverless Superman_ [Twitter Bot](https://www.raymondcamden.com/2017/10/20/upgrading-serverless-superman-to-ibm-composer/)
+* An app that [relays SMS to email](https://medium.com/openwhisk/a-composition-story-using-ibm-cloud-functions-to-relay-sms-to-email-d67fc65d29c) 
 
+## Feedback
 We welcome your feedback and criticism. Find bugs and we will squash
 them. And will be grateful for your help. As an early adopter, you
 will also be among the first to experience even more features planned
 for the weeks ahead. We look forward to your feedback and encourage
 you to [join us on slack](http://ibm.biz/composer-users).
-
-This repository includes:
-
```

Comment:
The index is useful - why remove it?

Reply:
I'll add it back. I pulled the `master` branch and thought the index was outdated (with `conductor` etc). This looks like an updated index.